qemu/tests/virtio_serial_file_transfer.py

```diff
@@ -155,7 +155,7 @@ def check_output(sender, output):
 
 @error_context.context_aware
 def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
-                  sender='both'):
+                  sender='both', host_clean=True):
```

Comment:
Please add doc string for it, and I will prefer name it `clean_file`, to handle guest file together.

Reply:
Thanks and update.